tendermint/src/lite/verifier.rs

```diff
@@ -261,29 +267,24 @@ pub fn verify_header<TS, SH, VS, L, S, R>(
 where
     TS: TrustedState<LastHeader = SH, ValidatorSet = VS>,
     L: TrustThreshold,
-    S: Store<SignedHeader = SH>,
+    S: Store<TrustedState = TS>,
     R: Requester<SignedHeader = SH, ValidatorSet = VS>,
     VS: ValidatorSet,
     SH: SignedHeader,
 {
     // Check if we already trusted a header at the given height and it didn't expire:
     if let Ok(sh2) = store.get(height) {
```

Comment:
We should probably change the `sh` to `ts` wherever signed header became trusted state.